README.md

```diff
@@ -0,0 +1 @@
+# Something great!
```

Comment:
@padznich please add more information like learning new css features or something like that.